lib/web/fotorama/fotorama.js

```diff
@@ -1140,7 +1140,24 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        el.addEventListener ? el.addEventListener(e, fn, !!bool) : el.attachEvent('on' + e, fn);
+        let supportsPassive = false;
+        try {
+            const options = {
+                get passive() {
+                    supportsPassive = true;
+                },
+            };
+
+            document.addEventListener('test', null, options);
+            document.removeEventListener('test', null, options);
+        } catch (err) {}
+
+        const options = (supportsPassive) ? {
+            passive: true
```

Comment:
It looks like we lost the `e === 'wheel'` change during a refactor. This is specifically called out in the pull request description, suggesting that it was an important part of the intended changes.
```suggestion
            passive: e === 'wheel',
```